src/PDS.API.Tests/DetailServiceTest.cs

```diff
@@ -0,0 +1,22 @@
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+using PDS.API.Services;
+
+namespace PDS.API.Tests;
+
+public class DetailServiceTests
+{
+
+    [Test]
+    public async Task GetDetailsByPackageKey()
+    {
+
+        var fixture = new InfrastructureFixture();
+        var detailService = fixture.ServiceProvider.GetService<IDetailService>();
```

Comment:
Please change the name from IDetailService to IDetailsService. You can change variable names as well.